packages/p2p/src/components/my-ads/ad-status.scss

```diff
@@ -1,15 +1,15 @@
 .ad-status {
     &--active {
-        background: rgba(75, 180, 179, 0.16);
+        background-color: var(--status-active);
         border-radius: 1.6rem;
         padding: 0.2rem 1.6rem;
         text-align: center;
         width: 7.4rem;
     }
 
     &--inactive {
-        border: 1px solid var(--status-danger);
-        border-radius: 0.2rem;
+        background-color: var(--status-in-active);
```

Comment:
`inactive` is one word, so this should be `--status-inactive`.

Reply:
Will make the change